developer-docs-site/docs/guides/nfts/aptos-token-overview.md

```diff
@@ -1,130 +1,70 @@
+
+Like Ethereum, Aptos requires indexing to determine the set of all NFTs owned by an account, while Solana has no need.
 
 ## Token standard comparison
-In Ethereum or even the whole blockchain world, the Fungible Token (FT) was initially introduced by
-[EIP-20](https://eips.ethereum.org/EIPS/eip-20), and Non-Fungible Token (NFT) was defined in 
-[EIP-721](https://eips.ethereum.org/EIPS/eip-721). Later, [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155)
-was proposed to combine FT and NFT or even Semi-Fungible Token (SFT) into the one standard. 
-
-One deficiency of the Ethereum token contract is each token having to deploy individual contract code onto
-a contract account to distinguish it from other tokens even if it simply differs by name.
-Solana account model enables another pattern where code can be reused so that one generic program operates on
-various data. To create a new token, you could create an account that can mint tokens and more accounts that can
-receive them. The mint account itself uniquely determines the token type instead of contract account, and these are all
-passed as arguments to the one contract deployed to some executable account.
-
-The Aptos token standard shares some similarities with Solana, especially how it covers FT, NFT and SFT in one standard and
-also has a similar generic token contract, which also implicitly defines token standard. Basically, instead of deploying
-a new ERC20 smart contract for each new token, all you need to do is call a function in the contract with necessary
-arguments. Depending on what function you call, the token contract will mint/transfer/burn/... tokens.
+
+The Fungible Token (FT) was initially introduced by [EIP-20](https://eips.ethereum.org/EIPS/eip-20), and Non-Fungible Token (NFT) was defined in [EIP-721](https://eips.ethereum.org/EIPS/eip-721). Later, [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155) combined FT and NFT or even Semi-Fungible Token (SFT) into one standard. 
+
+The Ethereum token standards requires each token to deploy their own individual contract code to distinguish collection of tokens. Solana account model enables another pattern where code can be reused so that one generic program operates on various data. To create a new token, you could create an account that can mint tokens and more accounts that can receive them. The mint account itself uniquely determines the token type instead of contract account, and these are all passed as arguments to the one contract deployed to some executable account.
+
+The collection of Aptos token standards shares some similarities with Solana, especially how it covers FT, NFT and SFT into a common code on-chain. Instead of deploying a new smart contract for each new token, a creator needs to call a function in the contract with necessary arguments. Depending on what function you call, the token contract will mint/transfer/burn/... tokens.
```

Comment:
```suggestion
The collection of Aptos token standards shares some similarities with Solana, especially how it covers FT, NFT and SFT into a common on-chain code. Instead of deploying a new smart contract for each new token, a creator calls a function in the contract with the necessary arguments. Depending on which function you call, the token contract will mint/transfer/burn/... tokens.
```